src/zkp/pilog.rs

```diff
@@ -364,11 +400,11 @@ mod tests {
             random_plusminus_by_size_with_minimum(&mut rng, ELL + EPSILON + 1, ELL + EPSILON)?;
 
         // Sampling x in the range 2^ELL should always succeed
-        random_paillier_log_proof(&mut rng, &x_small)?;
+        random_paillier_log_proof_verification(&mut rng, &x_small)?;
```

Comment:
nit: This is not on you but since we're here: I'd prefer an explicit assertion here over the current sneaky failure-via-?
```
assert!(random_paillier_log_proof_verifies(&mut rng, &x_small).is_ok());
```